hyperjumptech/monika
# Fix: Move Instatus to Notification Workspace and Register It as a Notification Channel

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Move Instatus to notification workspace and register It as a notification channel. Will resolve #1047. 

## How did you implement / how did you fix it  
1.  Move Instatus code to notification workspace
2. Register it in `channels` variable

## How to test  
1. Run Monika with Instatus notification
2. Make the website in your probe down (make sure it is your website)
3. Make it up again

## Notes
There is a bug where Monika sends two notification if the probe has 2 alerts. And it will only recover one. The bug appears in version 1.15.1, so I didn't solve it in this PR. 

## Video
https://github.com/hyperjumptech/monika/assets/15191978/e96aba8d-8371-48e7-bdc2-d63a7a2b1ae6
